code/game/objects/items/explosives/mine.dm

```diff
@@ -286,3 +286,162 @@
 	customizable = TRUE
 	matter = list("metal" = 3750)
 	has_blast_wave_dampener = TRUE
+
+
+
+/obj/item/explosive/mine/bounding
+	name = "\improper M5 Bounding Mine"
+	desc = "The M5 Bounding Mine is a landmine that propels itself upwards when the tripwire is set off. It explodes mid-air and either maims, kills, or fucks up anything standing in the blast zone. The 'Bouncing Betty' has seen limited use since the year 2150 due to the IFF sensors on the earlier models were prone to failure, resulting in friendly casualties. Be careful to not trip this or you're going to be ending up in a hospital bed full of shrapnel."
+	icon = 'icons/obj/items/weapons/grenade.dmi'
+	icon_state = "m5"
+	force = 5.0
+	w_class = SIZE_SMALL
+	// layer = MOB_HIDING_LAYER = 0.1 // geeves told me this was a good idea but it didnt work so it's gonna be like claymores lol
+	throwforce = 5.0
+	throw_range = 6
+	throw_speed = SPEED_VERY_FAST
+	unacidable = TRUE
+	flags_atom = FPRINT|CONDUCT
+	allowed_sensors = list(/obj/item/device/assembly/prox_sensor)
+	max_container_volume = 120
+	reaction_limits = list(	"max_ex_power" = 175,	"base_ex_falloff" = 75,	"max_ex_shards" = 32,
+							"max_fire_rad" = 5,		"max_fire_int" = 20,	"max_fire_dur" = 24,
+							"min_fire_rad" = 2,		"min_fire_int" = 3,		"min_fire_dur" = 3
+	)
+	angle = 360
+	use_dir = FALSE
+	var/obj/effect/mine_tripwire/bounding
+
+/obj/effect/mine_tripwire/bounding
+	name = "bouncing betty tripwire"
+	anchored = TRUE
+	mouse_opacity = 0
```

Comment:
Same as the variables above. If the parent already defines the variables then you don't need to redefine them unless you're changing them. Try and use what the parent variable has for you so we don't double things.

This also includes functions. Unless you need to change them don't override them.

Reply:
the latest push should have fixed everything else